builder/config_reader_test.go

```diff
@@ -108,30 +108,31 @@ func testConfig(t *testing.T, when spec.G, it spec.S) {
 				})
 			})
 
-			when("'uri' is misspelled as url", func() {
+			when("unknown buildpack key is present", func() {
 				it.Before(func() {
 					h.AssertNil(t, ioutil.WriteFile(builderConfigPath, []byte(`
-[buildpack]
+[[buildpacks]]
 url = "noop-buildpack.tgz"
 `), 0666))
 				})
 
-				it("returns errors when 'uri' is misspelled as 'url'", func() {
+				it("returns an error", func() {
 					_, _, err := builder.ReadConfig(builderConfigPath)
-					h.AssertError(t, err, "unknown configuration elements 'buildpack'")
+					h.AssertError(t, err, "unknown configuration element 'buildpacks.url'")
 				})
 			})
-			when("'buildpack' is misspelled", func() {
+
+			when("unknown array table is present", func() {
```

Comment:
Made this more consistent with https://github.com/buildpacks/pack/blob/30cb3dca7c746d1b21c00dc550f5b8219d3a7b9f/buildpackage/config_reader_test.go#L117.